src/index.js

```diff
@@ -1 +1,33 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import apiData from './modules/hit-api.js';
+import {
+  nameInput, refreshBtn, scoreInput, submitBtn,
+} from './modules/scores.js';
+
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/DIvC379OvteODU5bAfDN/scores';
+
+submitBtn.addEventListener('click', async (e) => {
+  e.preventDefault();
+  await apiData(url, { user: nameInput.value, score: scoreInput.value });
+  nameInput.value = '';
+  scoreInput.value = '';
+  const msg = document.getElementById('alart');
+  setTimeout(() => { msg.innerHTML = 'Score added successfully'; }, 1000);
+  setTimeout(() => { msg.innerHTML = ''; }, 4000);
+});
+
+const listItems = document.getElementsByClassName('list')[0];
+
+async function showScores() {
+  listItems.innerHTML = '';
+  const { result: scores } = await fetch(url)
+    .then((resp) => resp.json());
+  scores.forEach((result) => {
```

Comment:
- [ ] You seem to have addressed the issues from the previous reviewer although you have introduced the `.then` syntax for fetching from the API. please go back to using the `async/await` syntax. this is a requirement for the project.
- [ ] Make sure that the project remains working after you make the change